micro-rdk-installer/src/main.rs

```diff
@@ -331,3 +367,105 @@ fn main() -> Result<(), Error> {
     };
     Ok(())
 }
+
+fn update_app_image(args: &AppImageArgs) -> Result<(), Error> {
+    let config = Config::load().map_err(|err| Error::SerialConfigError(err.to_string()))?;
+
+    let dir = tempfile::tempdir().map_err(Error::FileError)?;
+    let tmp_old = dir.path().join("running-ptable.img");
+
+    log::info!("Retrieving running partition table");
+    let mut flasher =
+        connect(&args.connect_args, &config, false, false).map_err(|_| Error::FlashConnect)?;
+    flasher
+        .read_flash(
+            PARTITION_TABLE_ADDR.into(),
+            PARTITION_TABLE_SIZE.into(),
+            0x1000,
+            64,
```

Comment:
What is the origin of these values? Why are they correct? When, if ever, might they need to be updated? Let's add some comments or declare some named constants.

Reply:
they're default values used in `espflash`. will add comments about it